backend/src/apiserver/client_manager.go

```diff
@@ -50,6 +50,7 @@ const (
 	mysqlExtraParams       = "DBConfig.ExtraParams"
 	archiveLogFileName     = "ARCHIVE_CONFIG_LOG_FILE_NAME"
 	archiveLogPathPrefix   = "ARCHIVE_CONFIG_LOG_PATH_PREFIX"
+	dbConMaxLifetime       = "DBConfig.DBMaxLifetime"
```

Comment:
NIT: How about using the unit to declare this is for connection life time seconds?

For exmaple:

`dbConMaxLifetime = "DBConfig.DBMaxLifetimeSec"`

Reply:
Sounds good to me. CLA is also good now.

Reply:
Nit: there are two DB in the name, we can omit the second one?